### ISSUE TITLE
[Enhancement] Use default alpine docker image to run docker image picker

### ISSUE THREAD

**Issue by peterzhuamazon** [opensearch-project/opensearch-build#814]
### Is your feature request related to a problem? Please describe

https://github.com/opensearch-project/opensearch-build/commit/6bcdf1650565efadba33209f284b3806114115ce#
@peternied unblock this by using the ci-runner image to trigger docker image picker.
It works but sort of defeat the purpose of a docker image picker.

Not only the above section has this issue, in the same Jenkinsfile, the post steps are directly running on LINUX HOST not in docker container.

I suggest replace both sections with a light weight image like Alpine since this is just a few steps, no need to pull a full stack packaged 2G image for that.

Thanks.

### Describe the solution you'd like

See Above.

**Comment by bbarani** [comment]
@peterzhuamazon Can you provide an update for this issue?

**Comment by peterzhuamazon** [comment]
Hi @bbarani already done right here:
* https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch/distribution-build.jenkinsfile#L76
